.docker/base/Dockerfile

```diff
@@ -4,33 +4,47 @@ LABEL maintainer "Irina Spiridonova <irinasp@microsoft.com>; Darren Gehring <dar
 USER everest
 WORKDIR ${MYHOME}
 
-#PAT token should be provided in the docker run command or as a variable in VSTS build definition 
-ENV PAT $PAT
+ENV CI_LOGS ${MYHOME}"/ci-logs/"
+
+#PAT token and agent name should be provided in the docker run command when docker container is started 
 
 #software-properties-common package is needed for add-apt-repository
-#installing latest version of git
+#installing latest version of git and other packages
 RUN sudo apt-get install --yes software-properties-common
 RUN sudo add-apt-repository ppa:git-core/ppa
 RUN sudo apt-get update
-RUN sudo apt-get install git nuget
+RUN sudo apt-get install --yes \
+  git \
+  nuget \
+  openssh-client
+
+#copying the keys, elevating permissions and changing the key's owner for the key
```

Comment:
can you put a note as to where docker expects the id_rsa and id_rsa.pub files to be found? is it in the current directory from which you launch the `docker create` command?